plugins/rapid7_insightvm/komand_rapid7_insightvm/triggers/scan_completion/trigger.py

```diff
@@ -94,11 +85,13 @@ def get_results_from_latest_scan(self, params: dict, scan_id: int) -> List[Dict[
                 assistance=f"Exception returned was {error}",
             )
 
-        results_dict = {}
+        assets_list = []
+        vulnerability_list = []
         for row in csv_report:
-            results_dict.update(Util.filter_results(params, row, results_dict))
+            assets_list.append(Util.filter_results(params, row)[0])
```

Comment:
we should only need to call filter_results once to save having to perform the filter and building each list twice

```
new_assets, new_vulns = Util.filter_results(params, row)
assets_lists.append(new_assets)
assets_vulns.append(new_vulns)
```